# origin: kraxel (BitKeeper)
# cset: 1.2009.15.31 (2.6) key=423318cekH84kN_J7JItaVUrMfgvtQ
# URL: http://linux.bkbits.net:8080/linux-2.6/cset@423318cekH84kN_J7JItaVUrMfgvtQ
# inclusion: upstream
# descrition: [PATCH] v4l: tuner update
# revision date: Thu, 31 Mar 2005 18:05:13 +0900
#
# S rset: ChangeSet|1.2009.15.30..1.2009.15.31
# I rset: drivers/media/video/mt20xx.c|1.1..1.2
# I rset: drivers/media/video/tda8290.c|1.1..1.2
# I rset: include/media/tuner.h|1.17..1.18
# I rset: drivers/media/video/tuner-simple.c|1.1..1.2
#
# Key:
# S: Skipped  ChangeSet file only
# O: Original Followed by Updated
# U: Updated  Included with updated range of versions
# I: Included Included verbatim
# E: Excluded Excluded on request from user
# D: Deleted  Manually deleted by subsequent user edit
# R: Revised  Manually revised by subsequent user edit
#
#
# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/03/12 08:29:02-08:00 kraxel@bytesex.org 
#   [PATCH] v4l: tuner update
#   
#   Minor update for the tuner module:  Add some new entries,
#   fix a bug in the tda8290 driver.
#   
#   Signed-off-by: Gerd Knorr <kraxel@bytesex.org>
#   Signed-off-by: Andrew Morton <akpm@osdl.org>
#   Signed-off-by: Linus Torvalds <torvalds@osdl.org>
# 
# include/media/tuner.h
#   2005/03/11 12:32:18-08:00 kraxel@bytesex.org +4 -0
#   v4l: tuner update
# 
# drivers/media/video/tuner-simple.c
#   2005/03/11 12:32:18-08:00 kraxel@bytesex.org +8 -1
#   v4l: tuner update
# 
# drivers/media/video/tda8290.c
#   2005/03/11 12:32:18-08:00 kraxel@bytesex.org +2 -2
#   v4l: tuner update
# 
# drivers/media/video/mt20xx.c
#   2005/03/11 12:32:18-08:00 kraxel@bytesex.org +2 -1
#   v4l: tuner update
# 
#
===== drivers/media/video/mt20xx.c 1.1 vs 1.2 =====
--- 1.1/drivers/media/video/mt20xx.c	2005-03-09 02:55:28 +09:00
+++ 1.2/drivers/media/video/mt20xx.c	2005-03-12 05:32:18 +09:00
@@ -1,5 +1,5 @@
 /*
- * $Id: mt20xx.c,v 1.3 2005/02/15 15:59:35 kraxel Exp $
+ * $Id: mt20xx.c,v 1.4 2005/03/04 09:24:56 kraxel Exp $
  *
  * i2c tv tuner chip device driver
  * controls microtune tuners, mt2032 + mt2050 at the moment.
@@ -7,6 +7,7 @@
 #include <linux/delay.h>
 #include <linux/i2c.h>
 #include <linux/videodev.h>
+#include <linux/moduleparam.h>
 #include <media/tuner.h>
 
 /* ---------------------------------------------------------------------- */
===== drivers/media/video/tda8290.c 1.1 vs 1.2 =====
--- 1.1/drivers/media/video/tda8290.c	2005-03-09 02:55:29 +09:00
+++ 1.2/drivers/media/video/tda8290.c	2005-03-12 05:32:18 +09:00
@@ -1,5 +1,5 @@
 /*
- * $Id: tda8290.c,v 1.5 2005/02/15 15:59:35 kraxel Exp $
+ * $Id: tda8290.c,v 1.7 2005/03/07 12:01:51 kraxel Exp $
  *
  * i2c tv tuner chip device driver
  * controls the philips tda8290+75 tuner chip combo.
@@ -123,7 +123,7 @@ static int tda8290_tune(struct i2c_clien
 	struct i2c_msg easy_mode =
 		{ I2C_ADDR_TDA8290, 0, 2, t->i2c_easy_mode };
 	struct i2c_msg set_freq =
-		{ I2C_ADDR_TDA8290, 0, 8, t->i2c_set_freq  };
+		{ I2C_ADDR_TDA8275, 0, 8, t->i2c_set_freq  };
 
 	i2c_transfer(c->adapter, &easy_mode,      1);
 	i2c_transfer(c->adapter, i2c_msg_prolog, ARRAY_SIZE(i2c_msg_prolog));
===== include/media/tuner.h 1.17 vs 1.18 =====
--- 1.17/include/media/tuner.h	2005-03-08 13:41:40 +09:00
+++ 1.18/include/media/tuner.h	2005-03-12 05:32:18 +09:00
@@ -93,6 +93,10 @@
 #define TUNER_THOMSON_DTT7610    52
 #define TUNER_PHILIPS_FQ1286     53
 #define TUNER_PHILIPS_TDA8290    54
+#define TUNER_LG_PAL_TAPE        55    /* Hauppauge PVR-150 PAL */
+
+#define TUNER_PHILIPS_FQ1216AME_MK4 56 /* Hauppauge PVR-150 PAL */
+#define TUNER_PHILIPS_FQ1236A_MK4 57   /* Hauppauge PVR-500MCE NTSC */
 
 #define NOTUNER 0
 #define PAL     1	/* PAL_BG */
===== drivers/media/video/tuner-simple.c 1.1 vs 1.2 =====
--- 1.1/drivers/media/video/tuner-simple.c	2005-03-09 02:55:29 +09:00
+++ 1.2/drivers/media/video/tuner-simple.c	2005-03-12 05:32:18 +09:00
@@ -1,5 +1,5 @@
 /*
- * $Id: tuner-simple.c,v 1.4 2005/02/15 15:59:35 kraxel Exp $
+ * $Id: tuner-simple.c,v 1.10 2005/03/08 08:38:00 kraxel Exp $
  *
  * i2c tv tuner chip device driver
  * controls all those simple 4-control-bytes style tuners.
@@ -204,6 +204,13 @@ static struct tunertype tuners[] = {
 	  16*160.00,16*454.00,0x41,0x42,0x04,0x8e,940}, // UHF band untested
 	{ "tda8290+75", Philips,PAL|NTSC,
 	  /* see tda8290.c for details */ },
+	{ "LG PAL (TAPE series)", LGINNOTEK, PAL,
+          16*170.00, 16*450.00, 0x01,0x02,0x08,0xce,623},
+
+        { "Philips PAL/SECAM multi (FQ1216AME MK4)", Philips, PAL,
+          16*160.00,16*442.00,0x01,0x02,0x04,0xce,623 },
+        { "Philips FQ1236A MK4", Philips, NTSC,
+          16*160.00,16*442.00,0x01,0x02,0x04,0x8e,732 },
 
 };
 unsigned const int tuner_count = ARRAY_SIZE(tuners);
